<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>902</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>566</width>
    <height>568</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GEOSYS Options</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>860</y>
     <width>511</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="visit_geosys_label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>830</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://www.geosys.com/bridge/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Visit GEOSYS website to get your credentials&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="credentials_group">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>10</y>
     <width>519</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>User Credentials</string>
   </property>
   <widget class="QLabel" name="credentials_label">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>30</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Use valid credentials from GEOSYS. The credentials are required to get the access token which will be used for every Bridge API request.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="save_credentials_checkbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Credentials</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_form">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>112</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>371</width>
      <height>21</height>
     </size>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Bridge API password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_form">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>81</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>371</width>
      <height>21</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>Bridge API username</string>
    </property>
   </widget>
   <widget class="QLabel" name="username_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>81</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>112</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="client_id_form">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>140</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>371</width>
      <height>21</height>
     </size>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Client ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="client_id_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Client ID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="client_secret_form">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>170</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>371</width>
      <height>21</height>
     </size>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Client Secret</string>
    </property>
   </widget>
   <widget class="QLabel" name="client_secret_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Client Secret</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="region_group">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>250</y>
     <width>519</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Geographical Region</string>
   </property>
   <widget class="QRadioButton" name="us_radio_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>US (Fields located in USA, Canada and Australia.)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="eu_radio_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Europe (Fields located in Europe, South America and South Africa)</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="region_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose your region based on your fields geographical regions. This will decide which GEOSYS service would be used.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="testing_service_checkbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Use testing service</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="page_limit_group">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>570</y>
     <width>519</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Paging Limit</string>
   </property>
   <widget class="QLabel" name="page_limit_label_2">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>30</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set the paging limit for the maximum responses to be returned for each request. Larger limits will result in longer processing times and possible API timeouts.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="page_limit_label">
       <property name="text">
        <string>Max items per request</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="page_limit_form">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="coverage_parameters_group">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>420</y>
     <width>519</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Coverage Parameters</string>
   </property>
   <widget class="QLabel" name="sowing_date_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>111</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sowing Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="crop_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Crop Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="crop_combo_box">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>81</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
   </widget>
   <widget class="QDateEdit" name="sowing_date_edit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>111</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="coverage_parameters_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>35</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set the desired coverage parameters for each request. This parameters are required and will be applied to each coverage request</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="output_directory_group">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>690</y>
     <width>519</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Map Output Directory</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>25</number>
    </property>
    <property name="rightMargin">
     <number>25</number>
    </property>
    <item row="2" column="0">
     <widget class="QLineEdit" name="output_directory_form"/>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="output_directory_chooser">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="output_directory_label">
      <property name="text">
       <string>Set the output directory for map creation. Every map created by GEOSYS plugin will be placed in this directory.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>username_form</tabstop>
  <tabstop>password_form</tabstop>
  <tabstop>save_credentials_checkbox</tabstop>
  <tabstop>connect_button</tabstop>
  <tabstop>us_radio_button</tabstop>
  <tabstop>eu_radio_button</tabstop>
  <tabstop>testing_service_checkbox</tabstop>
  <tabstop>crop_combo_box</tabstop>
  <tabstop>sowing_date_edit</tabstop>
  <tabstop>page_limit_form</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
